concent_api/api_testing_common.py

```diff
@@ -455,6 +456,16 @@ def create_signed_report_computed_task(
                 task_to_compute=task_to_compute,
                 size=REPORT_COMPUTED_TASK_SIZE,
             ),
-            PROVIDER_PRIVATE_KEY,
+            provider_private_key,
         )
         return signed_message
+
+
+def get_cluster_name_from_address(address):
+    cluster_name = address.split('.')[0]
+    # If staging or testnet cluster will be tested then it should return 'staging' or 'test'
+    # elsewhere it should always return 'devel' because of possibility to test Concent locally
```

Comment:
That comment doesn't make much sense. So how do we test payments locally? We should be able to still do it in my opinion. You just need different `GNT_DEPOSIT_CONTRACT_ADDRESS` in local settings.

Reply:
We can still test it locally. If we give `http://locahost:8000` as address where you want to send request then it gets `devel` deposit address which is now used by all of us to tests.

Reply:
Yes, but we don't want to send e2e tests on localhost with GNT_DEPOSIT_ADDRES from dev, because it will cause `nonce to low` error, and we will need to clear database on dev to fix this.

Reply:
But for this moment we do not have more `GNT_DEPOSIT_ADDRESS`es so we need to use `devel` address locally, moreover `devel` is only testing cluster, if someone want to test anything on `devel` must be done new deploy and then database will be again initiated with new nonce. If in the same moment nobody will try to test it locally then it won't cause errors :)